src/System.CodeDom/tests/CodeGenerationTests.cs

```diff
@@ -191,4 +192,15 @@ private static string CoalesceWhitespace(string str)
             return sb.ToString();
         }
     }
+
+    public static class Common
+    {
+        public static IEnumerable<T> ToIEnumerable<T>(this IEnumerator<T> enumerator)
```

Comment:
Where is this used? I assume it's here for your upcoming collection tests or something?